test/functional/rpc_rawtransaction.py

```diff
@@ -141,21 +143,35 @@ def getrawtransaction_tests(self):
         for n in [0, 3]:
             self.log.info(f"Test getrawtransaction {'with' if n == 0 else 'without'} -txindex, with blockhash")
             # We should be able to get the raw transaction by providing the correct block
-            gottx = self.nodes[n].getrawtransaction(txid=tx, verbose=True, blockhash=block1)
-            assert_equal(gottx['txid'], tx)
-            assert_equal(gottx['in_active_chain'], True)
-            if n == 0:
-                self.log.info("Test getrawtransaction with -txindex, without blockhash: 'in_active_chain' should be absent")
-                gottx = self.nodes[n].getrawtransaction(txid=tx, verbose=True)
+            for v in [1, 2]: # verbosity level
+                gottx = self.nodes[n].getrawtransaction(txid=tx, verbose=v, blockhash=block1)
                 assert_equal(gottx['txid'], tx)
-                assert 'in_active_chain' not in gottx
-            else:
-                self.log.info("Test getrawtransaction without -txindex, without blockhash: expect the call to raise")
-                err_msg = (
-                    "No such mempool transaction. Use -txindex or provide a block hash to enable"
-                    " blockchain transaction queries. Use gettransaction for wallet transactions."
-                )
-                assert_raises_rpc_error(-5, err_msg, self.nodes[n].getrawtransaction, txid=tx, verbose=True)
+                assert_equal(gottx['in_active_chain'], True)
+                if v == 2:
+                    # fee is in verbosity 2 only
+                    assert_equal(gottx['fee'], Decimal('0.00002820'))
```

Comment:
What is supposed to guarantee this is correct?

Reply:
The value of fee is not important. I want to see that the field fee exists. Change it to `assert('fee' in gottx['fee'])` ?

There are another tests (outside the scope of this PR) which validate the fee amount.

Reply:
Maybe do `tx_fee = -self.nodes[2].gettransaction(tx)['fee']` immediately after sending and compare to that?

Reply:
No fee field in `gettransaction` . 
I added `assert('fee' in gottxt)`  but kept `assert_equal(gottx['fee'], Decimal('0.00002820'))`